VAMobile/ios/Podfile.lock

```diff
@@ -515,8 +515,6 @@ PODS:
     - React-Core
   - RNCAsyncStorage (1.18.1):
     - React-Core
-  - RNCClipboard (1.5.1):
-    - React-Core
```

Comment:
had to run pod install since I was dealing with some dependency stuff and it caught that this wasn't removed apparently in the pods.